agent/arcus-system/src/test/java/com/iris/agent/util/ByteUtilsTest.java

```diff
@@ -0,0 +1,261 @@
+package com.iris.agent.util;
```

Comment:
Can you add the apache 2 license header that's standard in the other java files?